AzureIotPnpDps.X/mqtt_packetPopulation/mqtt_iotprovisioning_packetPopulate.c

```diff
@@ -28,13 +28,14 @@ pf_MQTT_CLIENT pf_mqtt_iotprovisioning_client = {
   MQTT_CLIENT_iotprovisioning_receive,
   MQTT_CLIENT_iotprovisioning_connect,
   MQTT_CLIENT_iotprovisioning_subscribe,
-  MQTT_CLIENT_iotprovisioning_connected,  
+  MQTT_CLIENT_iotprovisioning_connected,
   NULL
 };
 
 extern const az_span device_model_id;
 extern uint8_t device_id_buf[100];
 extern az_span device_id;
+uint8_t atca_dps_id_scope[12];
```

Comment:
what is `atca`?

Reply:
I don't know what it stands for but it's Microchip's crypto library.